packages/design-system/src/components/InlineEditing/InlineEditing.spec.tsx

```diff
@@ -47,4 +47,15 @@ context('<InlineEditing />', () => {
 			.type('{enter}');
 		cy.getByTestId('inlineediting').contains('blah');
 	});
+
+	it('should not validate on Enter when multiline', () => {
+		cy.mount(<Textarea />);
+		cy.getByTestId('inlineediting.button.edit').click();
+		cy.getByTestId('ìnlineediting.textarea')
```

Comment:
Eh?
```suggestion
		cy.getByTestId('inlineediting.textarea')
```

Reply:
uh?
```suggestion
		cy.getByTestId('inlineediting.textarea')
```

Reply:
Good eye, didn't see the ì instead of i 👁️

Reply:
wow good catch, thanks
now, why does it works 😄